packages/ui-components/src/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FunctionComponent, ReactElement } from "react";
+import RCtooltip from "rc-tooltip";
+
+import "./Tooltip.scss";
+
+type TooltipPosition =
+  | "left"
+  | "right"
+  | "top"
+  | "bottom"
+  | "topLeft"
+  | "topRight"
+  | "bottomLeft"
+  | "bottomRight";
+
+type TooltipStyle = "default" | "light" | "dark";
+
+interface TooltipProps {
+  position?: TooltipPosition;
+  style?: TooltipStyle;
+  content: ReactElement | string;
```

Comment:
In developing the `Badge` component, there was some discussion that arose from [a comment](https://github.com/cockroachdb/ui/pull/3#discussion_r382102378) about a `content` prop vs `children`. We landed on using only `children` which makes the API for the component more intuitive. I was tempted at first to use a `content` prop also at first as it could allow for tighter control, but ultimately saw the wisdom is the flexibility of allowing the user to use child markup.

By removing the `content` prop it will also reduce the complexity of the component slightly as the code fork below in checking for content won't be required.

Reply:
we already use children, see an example:
`<Tooltip content={"tooltip text"}>
       <div>hover me</div>
 </Tooltip>`

Reply:
My fault. I just got the API of this component. `children` is the _target element_ of the Tooltip and the `content` is the content of the tooltip itself.